lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1247,8 +1247,12 @@ export class UserAgentApplication {
     if (this.getPostLogoutRedirectUri()) {
       logout = "post_logout_redirect_uri=" + encodeURIComponent(this.getPostLogoutRedirectUri());
     }
-    const urlNavigate = this.authority + "oauth2/v2.0/logout?" + logout;
-    this.promptUser(urlNavigate);
+    this.authorityInstance.resolveEndpointsAsync().then(a => {
+        const urlNavigate = a.EndSessionEndpoint
```

Comment:
can we use string interpolation her instead of concatenating?

Reply:
Done. 

I originally looked at the existing way of working. But I also like string interpolation more